resolvers/node/analyze.rs

```diff
@@ -383,7 +390,11 @@ impl<
               exports,
               Some(&referrer),
               ResolutionMode::Import,
-              conditions,
+              &ConditionResolver::new(ConditionResolverOptions {
+                conditions: conditions.to_vec(),
+                default_import_conditions: Vec::new(),
+                default_require_conditions: Vec::new(),
```

Comment:
See other comment below, but these two properties needing to be provided is an example of how `ConditionResolver` is not the right thing to be passing around here (instead, we can just pass conditions as before).